### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Rad-007** [smartcontractkit/full-blockchain-solidity-course-py#1544] [Q&A]
![image](https://user-images.githubusercontent.com/84834907/173909968-6ab33e0b-66aa-45ed-997e-b65349a03a64.png)

Help Anyone
No idea what is happening 
New to this

**Comment by cromewar** [comment]
Hello @Rad-007 
For me it seems it's related to your ganache installation, so may be this tips might help.
1. The new package for ganache is not anymore ganache-cli but just ganache.
2. Be sure to have ganache UI closed before running your script.
3. It's preferably to use a virtual python env for the web3 libraries.

Additionally please give a look to #980 and reformulate a little bit the question.
